test/specs/specification.spec.js

```diff
@@ -1035,7 +1079,7 @@ describe('Specification', function() {
             expect(invertedConversions[1]).property('offset').equal(-10);
         });
 
-        it('should work correctly with multiple conversions using a manual °C -> °F formula', function() {
+        it('should work correctly with multiple conversions using a manual °C -> °F formular', function() {
```

Comment:
Hey! 

I've got a nice surprise for you, I think you'll love it! Just take  a look <http://period.davidwoodruff.org/aebcuvl>

My Best, alan